assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -104,22 +115,62 @@ function MyStories() {
       searchTerm: typeaheadValue,
       sortDirection: viewStyle === VIEW_STYLE.LIST && currentListSortDirection,
       status,
+      page: currentPage,
     });
   }, [
     viewStyle,
     currentListSortDirection,
+    currentPage,
     currentStorySort,
     fetchStories,
     status,
     typeaheadValue,
   ]);
 
-  const filteredStories = useMemo(() => {
-    return stories.filter((story) => {
-      const lowerTypeaheadValue = typeaheadValue.toLowerCase();
-      return story.title.toLowerCase().includes(lowerTypeaheadValue);
+  const setCurrentPageClamped = useCallback(
```

Comment:
Hmmmmm.... I might just be missing it, but I'm not seeing where `setCurrentPageClamped` is being called with a `function` parameter... do we need this `typeof` check?  Is that for a future PR or am I just missing something obvious? 🤔

Reply:
yea good point. no need to support it if we're not using it

Reply:
I removed it. I'd copied it from a suggestion max made earlier and figured it wouldn't hurt to be kept in but it's added complexity we never need.